cmd/kubeadm/app/cmd/phases/etcd.go

```diff
@@ -53,20 +53,30 @@ func NewEtcdPhase() workflow.Phase {
 		Phases: []workflow.Phase{
 			newEtcdLocalSubPhase(),
 		},
+		CmdFlags: getEtcdPhaseFlags(),
 	}
 	return phase
 }
 
 func newEtcdLocalSubPhase() workflow.Phase {
 	phase := workflow.Phase{
-		Name:    "local",
-		Short:   "Generates the static Pod manifest file for a local, single-node local etcd instance.",
-		Example: etcdLocalExample,
-		Run:     runEtcdPhaseLocal(),
+		Name:     "local",
+		Short:    "Generates the static Pod manifest file for a local, single-node local etcd instance.",
+		Example:  etcdLocalExample,
+		Run:      runEtcdPhaseLocal(),
+		CmdFlags: getEtcdPhaseFlags(),
 	}
 	return phase
 }
 
+func getEtcdPhaseFlags() []string {
```

Comment:
Maybe we can define this flags on `newEtcdLocalSubPhase`

Reply:
this method is used both in `newEtcdLocalSubPhase` and in `NewEtcdPhase`.